onet-parse/src/fetch-data.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * This is a script that parses ONET web services endpoints to create
+ * INSERT statements that will be ingested into our database.
+ */
+import axios from 'axios';
+
+import { ONETJobDetailResponse, ONETStemJobResponse } from './types';
+
+const axiosOptions = {
+    mode: 'cors',
+    auth: {
+        username: 'raythked',
+        password: '5638nqd'
+    },
+    headers: {
+        'Accept': 'application/json'
+    }
+};
+
+/** 
+ * this one gets a list of all stem jobs, returns the code (ONET SOC CODE) 
+ * https://services.onetcenter.org/reference/online/browse/browse_stem
+*/
+const fetchStemJobs = async (): Promise<ONETStemJobResponse> => {
+    const promise = await axios.get('https://services.onetcenter.org/ws/online/stem_occupations/all?start=1&end=2000', {
```

Comment:
is that just grabbing the first 2000 occupations?

Reply:
there's actually only ~187 but I just put in a big number as a catch all in case for some reason there are more in the future. It's very hardcoded and rough and the other solution was to hit the endpoint once to grab the total and then hit it again with the actual total supplied as the `end` query param.
The other solution is the endpoint returns the next batch of paginated endpoints (by 20 entries at a time) and I could repeatedly call the endpoint until I get to the end.
ONET web service endpoint design is kinda gross... and the documentation is horrible